be/src/exprs/vectorized/json_functions.cpp

```diff
@@ -327,6 +331,32 @@ ColumnPtr JsonFunctions::_string_json(FunctionContext* context, const Columns& c
     return result.build(ColumnHelper::is_all_const(columns));
 }
 
+ColumnPtr JsonFunctions::_unescape_and_unquote_string(FunctionContext* context, const Columns& columns) {
```

Comment:
In this way the column needs to be copied, could you embed the `unescape` procedure into the `json_string` directly to eliminate this overhead?

Reply:
Maybe I think it's better to add a function with name `_json_string_unescaped` and not to change the behaviour of `json_string`.